src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LOGO = " ________      ________ _____  \n" +
+            "|  ____\\ \\    / /  ____|  __ \\ \n" +
+            "| |__   \\ \\  / /| |__  | |__) |\n" +
+            "|  __|   \\ \\/ / |  __| |  _  / \n" +
+            "| |____   \\  /  | |____| | \\ \\ \n" +
+            "|______|   \\/   |______|_|  \\_\\";
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    private static final String[] COMMANDS = {"help", "list", "todo", "deadline", "event",
+        "mark", "unmark", "bye"};
+
+    private static final int MAX_TASK = 100;
+    private static Task[] tasks = new Task[MAX_TASK];
+    private static int taskCount = 0;
+
+    private static void greet() {
+        String MESSAGE = "Hello! I'm Ever\n" +
```

Comment:
you can consider making this a final string so that there will not be multiple creations of the same string (although I understand the function is only called once)